### PR TITLE
Update table to remove deprecated Xcode versions

### PR DESCRIPTION
**Pull request by BytesGuy**
These images have been removed from CircleCI so need to be removed for our docs

### CODE REVIEW

File: jekyll/_cci2/testing-ios.md [hunk 1]
```diff
@@ -68,18 +68,6 @@ Running or testing Apple Silicon apps natively is currently not possible as Circ
...
  `10.3.0` | Xcode 10.3 (10G8)         | 10.14.4 | No | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1925/index.html) | [Release Notes](https://discuss.circleci.com/t/xcode-10-3-image-released/31561)
- `10.2.1` | * Xcode 10.2.1 (10E1001)    | 10.14.4 | No | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1911/index.html) | [Release Notes](https://discuss.circleci.com/t/xcode-10-2-1-image-released/30198)
- `10.1.0` | * Xcode 10.1 (10B61)        | 10.13.6 | No | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1901/index.html) | [Release Notes](https://discuss.circleci.com/t/xcode-10-1-image-released/26350)
- `10.0.0` | * Xcode 10.0 (10A255)       | 10.13.6 | No | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1893/index.html) | [Release Notes](https://discuss.circleci.com/t/xcode-10-0-gm-image-released/25202)
- `9.4.1`  | * Xcode 9.4.1 (9F2000)      | 10.13.3 | No | [Installed software](https://circle-macos-docs.s3.amazonaws.com/image-manifest/v1881/index.html)
-{: class="table table-striped"}
```
Comment (felicianotech) [inline jekyll/_cci2/testing-ios.md:75]: Is this line supposed to be removed?

Comment (BytesGuy) [reply jekyll/_cci2/testing-ios.md:75]: Nope, good catch, thanks!

File: jekyll/_cci2/testing-ios.md [hunk 2]
```diff
@@ -68,18 +68,7 @@ Running or testing Apple Silicon apps natively is currently not possible as Circ
...
-
-**Note:** The following images are deprecated and scheduled for removal on 14 September 2021. Please migrate away from these images as soon as possible. For more information, please see the [announcement thread on our forum](https://discuss.circleci.com/t/xcode-deprecation-notice-9-4-1-10-0-0-10-1-0-and-10-2-1/40515).
-
-* Xcode 9.4.1
-* Xcode 10.0.0
-* Xcode 10.1.0
-* Xcode 10.2.1
+ {: class="table table-striped"}
```
Comment (felicianotech) [inline jekyll/_cci2/testing-ios.md:71]: There's a leading space here. I'm not sure if that's okay.

**Review by felicianotech — APPROVED** [review]
Aside from the leading space question, looks good to me.

**Pull request merged**